timesketch/api/v1/resources/analysis.py

```diff
@@ -137,25 +179,18 @@ def post(self, sketch_id):
                 HTTP_STATUS_CODE_FORBIDDEN,
                 'Unable to run an analyzer without any data submitted.')
 
-        timeline_id = form.get('timeline_id')
-        if not timeline_id:
+        timeline_ids = form.get('timeline_ids')
```

Comment:
if you are changing this, you'll need to change the api_client/python.../timeline.py file as well, there is a function there called `run_analyzers` that needs to changed, line 236 from

```
'timeline_id': self.id,
```

to

```
'timeline_ids': [self.id],
```
otherwise the API client will not work anymore.